### ISSUE TITLE
Reading telegrams from P1

### ISSUE THREAD

**Issue by Mausy5043** [homewizard/python-homewizard-energy#430]
This code:
```(python)    
async with HomeWizardEnergyV1(host=IP_ADDRESS) as api:
    wiz_data = await api.data()
    print("\ndata\n{wiz_data}")
```
..returns the data from the P1-meter.
I'd like to use similar code to inspect the most recent telegram received from the P1-meter. For example using `await api.telegram()`.

Are there any plans to add this functionality?

**Comment by Mausy5043** [comment]
@DCSBL : I might give it a try. Do you want me to create a PR agains the `main`-branch?  

My own P1-meter is still on the API/v1. Is that okay? 

And how is that change then later migrated automagically to the API/v2?

**Comment by DCSBL** [comment]
We can add you to the v2 release group if you wish. You can fill in that form here: https://forms.gle/yYyKJYexaJv1fdae6

>  Do you want me to create a PR agains the main-branch?

Yes please

**Comment by Mausy5043** [comment]
I'm a bit hesitant, as I don't want to FUBAR my current set-up. Is the API/v2 backwards compatible with calls to the API/v1 ?

**Comment by DCSBL** [comment]
V1 and V2 live side-to-side

https://api-documentation.homewizard.com/docs/introduction#devices

**Comment by DCSBL** [comment]
But you can start with v1 and blindly see what the v2 implementation would look like, I can finetune the last pat IRL.

**Comment by Mausy5043** [comment]
Ah, okay. So I can acces both API/v1 and API/v2 in parallel. Using API/v1 for the "production" set-up and API/v2 for devlopement/testing on the same device.

**Comment by DCSBL** [comment]
Yes, v1 and v2 work at the same time for the P1 Meter. You _can_ disable v1 via the v2 API if you wish but that is up to you.

Can you refer to any documentation/message where you got the idea that you can only use one version? We might have to tweak that if that is what people think ;)

**Comment by Mausy5043** [comment]
I've created PR#491 to fix this proposal. I've tested it with the API/v1. Not tested with API/v2, but I expect no issues with that implementation as it follows the other methods closely.

**Comment by Mausy5043** [comment]
@DCSBL 
> Can you refer to any documentation/message where you got the idea that you can only use one version? We might have to tweak that if that is what people think ;)

It's probably just me. But, I would have felt more confident if the documentation would be a bit more reassuring. For example, you might want to consider changing the first sentence on [this page](https://api-documentation.homewizard.com/docs/category/api-v2) to something like:
